### PR TITLE
Tables

### PR DESCRIPTION
**Pull request by gcstaublin**
## What this did

Creates base table styles

- [ ] Base table
2016-01-06 02 27 06 pm

- [ ] Bordered table
2016-01-06 02 27 19 pm

- [ ] Striped and hoverable
2016-01-06 02 27 33 pm

- [ ] Colspan table
2016-01-06 02 27 47 pm

- [ ] RWD table
2016-01-06 02 28 09 pm

### CODE REVIEW

File: static/scss/_tables.scss [hunk 1]
```diff
@@ -31,85 +17,202 @@ $table_bdr: 1px solid $ui_gray_medium_color;
...
+  white-space: nowrap;
 }
 
 
-.#{$css-prefix}table__td, %table__td {
-  padding: 10px 8px;
+.#{$css-prefix}table__td {
+  padding: 15px 8px;
+  white-space: nowrap;
   // border-bottom: $table_bdr;
```
Comment (mfouquet) [inline static/scss/_tables.scss]: Not sure if you want to clean up some of the commented out code in here. Seems to be only a few lines.

File: static/scss/_tables.scss [hunk 2]
```diff
@@ -31,85 +17,202 @@ $table_bdr: 1px solid $ui_gray_medium_color;
...
+// --------------------------------------------
+@media only screen and (max-width: 960px) {
+
+  .#{$css-prefix}table--rwd { margin-bottom: 0; }
+
+  .#{$css-prefix}table--pinned {
+    position: absolute;
+    left: 0;
+    top: 0;
+    background: #fff;
```
Comment (mfouquet) [inline static/scss/_tables.scss]: Perhaps make this a variable?

Comment (gcstaublin) [inline static/scss/_tables.scss]: The breakpoint?

It probably should be ... but we'll also have different points where we'd want a table to become responsive, too.

Comment (mfouquet) [inline static/scss/_tables.scss]: Nope, the background color.

File: static/scss/_tables.scss (hunk 1 above)
Comment (gcstaublin) [inline static/scss/_tables.scss]: Most of that can probably come out at this point. I don't think there's any 'we might reverse course' stuff left in there.

File: static/scss/_tables.scss (hunk 2 above)
Comment (gcstaublin) [inline static/scss/_tables.scss]: :+1:

File: static/js/main.js [hunk 3]
```diff
@@ -41,4 +19,130 @@ $(function() {
...
+
+
+  $(window).load(updateTables);
+  $(window).on("redraw",function(){switched=false;updateTables();}); // An event to listen for
+  $(window).on("resize", updateTables);
+
+  // Split Table
+  function splitTable(original)
+  {
+    original.wrap("<div class='ko-table__wrapper' />");
```
Comment (mfouquet) [inline static/js/main.js]: Might be good to make variables out of these (ko-table-wrapper, ko-table--pinned) in case updates are made to the names later.

**Pull request merged**